### PR TITLE
addons: Python3 changes

### PR DESCRIPTION
**Pull request by MilhouseVH**
Not build or run-time tested - there could be more lurking issues that need fixing.

### CODE REVIEW

File: packages/addons/service/docker/source/default.py [hunk 1]
```diff
@@ -187,56 +188,56 @@ def print_notification(json_data):
...
+                                    __addon__.getLocalizedString(30012),
+                                    __addon__.getLocalizedString(event_string)])
 
     dialog = xbmcgui.Dialog()
     try:
         if message is not '':
             length = int(__addon__.getSetting('notification_length')) * 1000
             dialog.notification('Docker', message, '/storage/.kodi/addons/service.system.docker/resources/icon.png', length)
-            xbmc.log('## service.system.docker ## ' + unicode(message))
+            xbmc.log('## service.system.docker ## %s' % message)
```
Comment (queeup) [inline packages/addons/service/docker/source/default.py]: Maybe you can use .format for this for python3?
```python
xbmc.log('## service.system.docker ## {}'.format(message))
```

Comment (MilhouseVH) [reply packages/addons/service/docker/source/default.py]: Could do, I guess I'm a bit old skool. :smile:

Comment (MilhouseVH) [reply packages/addons/service/docker/source/default.py]: We could also use:
```
xbmc.log(f'## service.system.docker ## {message}')
```
There are plenty of ways to skin the string concatenation cat in Python 3.6+ - it all comes down to personal preference, and I suppose I tend to go with the most basic solution. :smile:

Comment (queeup) [reply packages/addons/service/docker/source/default.py]: Basic is always good :smile:

**Comment by MilhouseVH** [comment]
Might as well merge these, then maybe they can be tested.

**Pull request merged**